contributed_definitions/nyaml/NXion.yaml

```
@@ -74,21 +69,18 @@ NXion(NXobject):
     doc: |
       Signed charge state of the ion in multiples of electron charge.
       
-      Only positive values will be measured in atom probe microscopy as the
-      ions are accelerated by a negatively signed bias electric field.
+      Only positive values will be measured in atom probe microscopy
```

This seems very focused on APM, but the NXion class is much more general, no? As you say in the class docstring, it can also be used for other mass spectrometries? Could this either go to the NXapm appdef or could a sentence be added here to say that this text is just one example of how to use `charge`.

---

You are right, currently the usage is focused heavily on atom probe because there are examples because of which stuff was implementable, a few months ago I mentioned that I back up what is here also your idea make such base class specific for talking about ions in general, no feedback coming from anybody, neither B nor A so for now we should merge the class as it is but revive the specific harmonization between chemicals as a base class used in NOMAD by e.g. Area A and a similar conceptual construct which is however used and needs for irrespective the discussions within NOMAD customizations for charged ions as are proposed here.

---

Ok, makes sense to discuss this later.

However, in the particular line above, I would suggest to make clearer that this is just for the example of APM. Maybe:
```suggestion
In the example of atom probe microscopy, only positive values will be measured as the ions are accelerated by a negatively signed bias electric field.

In the case that the charge state is not explicitly recoverable, the value should be set to zero. This is for example the case in atom probe microscopy when using classical ranging definition files in formats like RNG, RRNG. These file formats ..
```

Like this, we can keep this example also later when `NXion` is used in other contexts.

---

Agree, changed it according to your suggestion, thanks